src/index.ts

```diff
@@ -264,6 +264,7 @@ export function main(argString = ""): void {
   }
 
   const args = argString.split(" ");
+  let simulateDiet = false;
```

Comment:
put this in globalOptions

Reply:
I guess it's probably temporary for diagnostic stuff, so this is fine.

But if you want it to be a permanent feature, or if you are going to transform this architecture into `nodiet`, make it part of globalOptions.

Reply:
why not just put the runDiet call inside the if statement? I don't think any other args will interact with this one